exts/cesium.omniverse/cesium/omniverse/ui/attributes/ion_raster_overlay_attributes_widget.py

```diff
@@ -25,11 +25,8 @@ def _customize_props_layout(self, props):
                 CustomLayoutProperty("cesium:ionAssetId")
                 CustomLayoutProperty("cesium:ionAccessToken")
                 CustomLayoutProperty("cesium:ionServerBinding")
-            with CustomLayoutGroup("Rendering"):
-                CustomLayoutProperty("cesium:alpha", build_fn=build_slider(0, 1))
                 CustomLayoutProperty("cesium:overlayRenderMethod")
```

Comment:
Should `cesium:overlayRenderMethod` be added to `build_common_raster_overlay_properties` and removed from here?

Reply:
Thinking about it...yes. I moved it there. It's only in `ion_raster_overlay_attributes_widget` because it's currently less than useful for polygon raster overlays and probably will never make sense for WMS and related raster overlays.

So I added a function param where it's suppressed by default. To get it in the widget, you need to do this:

```
        with frame:
            ...
            build_common_raster_overlay_properties(add_overlay_render_method=True)
```
It's still of course under "Raw USD Properties." I'm not sure if that can be suppressed. But it being only there hopefully communicates that it's not a property we're currently handling.